plugins/netskope/icon_netskope/actions/get_all_url_list/action.py

```diff
@@ -14,7 +14,7 @@ def __init__(self):
 
     def run(self, params={}):
         request_params = (
-            {"pending": self._check_pending(params.get(Input.STATUS))} if params.get(Input.STATUS) else None
+            {"pending": self._check_pending(params.get(Input.STATUS))} if params.get(Input.STATUS) != "any" else None
```

Comment:
Don't entirely know the context but would it make more sense for the if statement to be `if params.get(Input.STATUS) == "pending"`? As I'm not sure if you want to set the pending object if the Input.STATUS == "applied"?

Reply:
@tcheng-r7 applied and pending are filters for the api. In UI they are displayed either or. So, we added any which will not pass any filter params. So, in future if there is a new state, Users can use this to get new state and continue with their workflows with minimal modifications.